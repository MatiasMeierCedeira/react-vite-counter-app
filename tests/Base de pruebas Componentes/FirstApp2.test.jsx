import { render,screen } from "@testing-library/react"
import { FirstApp } from "../../src/FirstApp";


    describe('Pruebas First App',()=>
        {

          const title ='Soy el Maty'; 

           test('Should make a match wiht the snapshot',()=>
            {
              render(<FirstApp title = {title} />)
              expect(screen.getByText).toBeTruthy();
            })

            test('Should make a match wiht the H1',()=>
              {
                render(<FirstApp title = {title} />)
                expect(screen.getByRole('heading',{level:1}).innerHTML).toContain(title);
              })

        })