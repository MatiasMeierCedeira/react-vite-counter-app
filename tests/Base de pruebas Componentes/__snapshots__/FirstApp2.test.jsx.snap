// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas First App Should make a match wiht the snapshot 1`] = `
<div>
  <h1>
    Soy el Maty
  </h1>
  <p>
    Fernando Herrera
  </p>
</div>
`;

exports[`Pruebas First App Should make a match wiht the snapshot: Soy el Maty 1`] = `
<div>
  <h1>
    Soy el Maty
  </h1>
  <p>
    Fernando Herrera
  </p>
</div>
`;

exports[`Pruebas First App Validaci√≥n de parametros con snapshot 1`] = `
<div>
  <h1>
    Soy el Maty
  </h1>
  <p>
    Fernando Herrera
  </p>
</div>
`;
